App.js

```diff
@@ -1,11 +1,25 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View, Button } from 'react-native';
```

Comment:
Don't change the contents of the `App.js` file - this file shouldn't be changed when making separate components or screens